src/app_service/common/social_links.nim

```diff
@@ -11,13 +11,24 @@ type
   SocialLink* = object
     text*: string
     url*: string
+    icon*: string
 
   SocialLinks* = seq[SocialLink]
 
+proc socialLinkTextToIcon(text: string): string =
```

Comment:
I am not sure if this is a proper place for this. These icon names are strictly UI related. Wondering what was wrong with the previous approach (other than duplication which could be generalized in qml as well)?

Reply:
Yeah duplication and as these icons seem "static", I thought the backend would be a good place for it; I can definitely move them back to QML to some shared location.

Reply:
This is fine for me as long as we don't do the same stuff both in Nim and QML

Reply:
Defining icon names on the backend side feels just counterintuitive. It is nothing crucial, feel free to leave it as is or move it back to QML, up to you.